index.js

```diff
@@ -18,6 +19,28 @@ const TimerMixin = require('react-timer-mixin');
 const DefaultTabBar = require('./DefaultTabBar');
 const ScrollableTabBar = require('./ScrollableTabBar');
 
+class StaticContainer extends Component{
```

Comment:
It's almost the same. I'll change it. Appreciate if you can give feedback on other part of logic and test it's effectiveness